### DISCUSSION TITLE
cubase 11 renders with audio gridder ghosting /PDC issue

### DISCUSSION THREAD

**Discussion by norburybrook** [apohl79/audiogridder#519] [Help] [answered]
I've noticed recently now I'm using audio gridder fully that I'm getting what seem like PDC issues when bouncing from cubase 11. I can sometimes hear a pre reverb for example as if the PDC is running ahead of the track.

Anyone else? it's not 100% and it seems to be on any gridder plugin. I had it on a bass drum yesterday with an AA channel strip, I had ghosting of the kick drum.
I first noticed it on a a reverb and thought it might be because it was from a jbridge powercore but it's not that.

I'm on 1.1 
M

**Comment by norburybrook** [comment]
Also whenever a render is finished I can see that the server quits and restarts , why is this?

M

**Comment by norburybrook** [comment]
upgraded to 1.2 beta and still the same. I'm getting poor timing from rendering mixes, not sure what's happened recently as I've not noticed before . Is there anyway to help test this? it's not a fixed issue, sometimes it's Ok .

**Comment by CaBuffalo** [comment]
Hi, Norburybrook,

Please follow the instruction below to report the bug. Thanks.

https://audiogridder.com/bug-reports/

On Sun, Apr 25, 2021 at 9:09 AM norburybrook ***@***.***>
wrote:

> Ok after some more testing on just a kick drum loop it seems that this is
> onlpy happening when rendering to an Mp3 file. Rendering to a wav file is
> fine. I'm not sure what the difference is though . I render client mixes to
> Mp3 so this is important.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/519#discussioncomment-656013>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6D67RM5YFKMD3FKXBLTKQ5CBANCNFSM43RNHD3Q>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by norburybrook** [comment, marked as answer]
more testing and it's directly associated with the Gridder buffer size. The ghosting is directly related to the buffer size so setting it to 30 you can hear the delay really clearly on the rendered file.

edit: I think I've solved this. My poco plugins were showing as dissabled. I've rebooted the server and everything seems OK. Looks like the POCO plugins were causing some kind of PDC issue when rendering 

M

**Reply by apohl79** [reply]
thx for sorting this out! :-)
